lib/jets/cfn/upload.rb

```diff
@@ -90,8 +90,21 @@ def upload_to_s3(full_path)
 
       key = s3_key(full_path)
       obj = s3_resource.bucket(bucket_name).object(key)
-      puts "Uploading s3://#{bucket_name}/#{key}" # uncomment to see and debug
-      obj.upload_file(full_path, acl: "public-read", cache_control: cache_control)
+      puts "Uploading and setting content type for s3://#{bucket_name}/#{key}" # uncomment to see and debug
+      obj.upload_file(full_path, { acl: "public-read", cache_control: cache_control }.merge(guess_content_type_headers(full_path)))
+    end
+
+    def guess_content_type_headers(full_path)
+      case File.extname(full_path)
```

Comment:
Let’s clean this up with a map Hash that holds the mapping. Can also get rid of the guess_ from the method name.